crates/relayer-types/src/core/ics24_host/identifier.rs

```diff
@@ -292,6 +292,50 @@ impl PartialEq<str> for ConnectionId {
     }
 }
 
+// A Vec<ConnectionId> wrapper for the multi-hop channels PoC implementation
+#[derive(Clone, Debug, Default, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+pub struct ConnectionIds(Vec<ConnectionId>);
+
+impl ConnectionIds {
+    pub fn as_slice(&self) -> &[ConnectionId] {
+        &self.0
+    }
+
+    pub fn into_vec(self) -> Vec<ConnectionId> {
+        self.0
+    }
+}
+
+impl FromStr for ConnectionIds {
+    type Err = ValidationError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let connection_ids = s
+            .split('/')
+            .map(|conn_id| conn_id.parse())
+            .collect::<Result<Vec<ConnectionId>, _>>()?;
+
+        if connection_ids.is_empty() {
+            return Err(ValidationError::empty());
```

Comment:
Let's define a new error variant for this, perhaps something like `EmptyConnectionHops` with a message like `cannot parse empty list of connection ids into connection hops`